command/read.go

```diff
@@ -0,0 +1,61 @@
+package command
+
+import (
+	"io"
+	"strings"
+
+	"github.com/gotify/cli/v2/utils"
+)
+
+func readMessage(args []string, r io.Reader, output chan<- string, split *rune) {
```

Comment:
`output` could be a callback like 
```
func readMessage(args []string, r io.Reader, output func(string), split *rune) {
```
then we wouldn't need the goroutine and channel. What do you think?

Reply:
Yeah it has pros and cons, I did it with a channel so that all the error handling code could stay at the caller side and readMessage() only takes care of IO